Utils/action_utils_wait.json

```
@@ -0,0 +1,25 @@
+{
+    "arguments": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "properties": {
+            "duration": {
+                "description": "Time to wait, in seconds",
+                "type": "integer"
```

We can add [a minimum and a maximum](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-validation-keywords-for-num) to force the input of the customer
```suggestion
                "description": "Time to wait, in seconds",
                "type": "integer",
                "minimum": 0,
                "maximum": 3600
```